libs/external-db-spanner/src/sql_schema_translator.ts

```diff
@@ -91,12 +91,28 @@ export default class SchemaColumnTranslator implements ISpannerSchemaColumnTrans
                 return `STRING${this.parseLength(2 ** 16)}`
```

Comment:
I know it's not related to this pr, and that you didn't set that, but maybe we should consider the size of the string types here, 
text_small = 2**8 (which is reasonable)
text_medium = 2**16 (65,536 chars) - which is not medium in any way - I think it should be something around 2**12 (4,096) 
text_large = 2**32 (59125081250218501850218590852109581 chars approx) - which is extremely large (specifically when we are talking about `text_language`), I think we should go with around 2**20 or something close to that.

Reply:
This change is not directly related to this PR, and the PR is already quite large. I suggest addressing this change in a separate PR to maintain a clear scope for this PR.

Reply:
np, wrote it because I've already been there, you are welcome to do it in another PR, shouldn't cause any conflicts obviously